Group rules by type in actions config page

djbingham opened the pull request:
https://eaflood.atlassian.net/browse/FCEP-86

jaucourt commented on server/views/actions-rule-set.njk:
```diff
@@ -0,0 +1,56 @@
+    {% endif %}
+
+    <div id={{ ruleID }} class="govuk-summary-list__row" style="{{ backgroundColour }}">
+      <dt class="govuk-summary-list__key" style="padding-left: 10px">
```
There are several places where you've set left padding to be 10px, would it be worth creating a new class?

djbingham replied:
That's all just moved from `actions.njk` into a separate file so it could be reused. I didn't even look at the content of the moved code!

Seems reasonable to create a class, although probably not worth fussing over at this stage?

jaucourt replied:
No, I felt it was a bit of a pettifogging point when I made it, so leave it for now :)

jaucourt commented on test/services/actionsService.test.js:
```diff
@@ -1,64 +1,117 @@
-    expect(result.length).toEqual(1)
-    expect(result[0]).toEqual(mockActionServiceResult.actions[0])
+  afterAll(() => {
+    jest.unmock('@hapi/wreck')
```
I don't think there's any need for this; jest will automatically remove mocks after exiting a test file. This actually marks that a module should not be mocked, I think it'd be useful in the context of doing auto mocking.

djbingham replied:
Yes, my mistake. I was copying the way we've generally done tests so far but you are correct. The `unmock` function is intended for disabling a mock for a single test or a subset of tests within a file, not for removing mocks at the end.

I'll take that out.

djbingham replied:
The `afterAll` has now been removed.

djbingham commented:
In addition to addressing the comment above regarding `jest.unmock`, I've fixed the widths of rule groups to 100% in line with the comment on [Jira](https://eaflood.atlassian.net/browse/FCEP-86).

I also applied default govuk styling to the heading above each block of rules via the CSS class `govuk-heading-m`, which is recommended for `<h3>` elements.
